packages/test/version-test-1/src/main.tsx

```diff
@@ -49,12 +49,56 @@ export class VersionTest extends DataObject implements IFluidHTMLView {
         });
     }
 
-    public render(div: HTMLElement) {
-        const rerender = () => {
+    public async render(div: HTMLElement) {
```

Comment:
It would be great to open a work item for Mark to incorporate blob payload into outer (stress) tests he is adding